actionview-component.gemspec

```diff
@@ -43,4 +43,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "better_html", "~> 1"
   spec.add_development_dependency "rubocop", "= 0.74"
   spec.add_development_dependency "rubocop-github", "~> 0.13.0"
+
+  spec.post_install_message = "actionview-component has been renamed to view_component, and will no longer be published in this namespace. Please update your Gemfile to use view_component."
```

Comment:
Do you think it'd be worth including a link to `https://github.com/github/view_component/issues/206` (https://github.com/github/view_component/issues/206) in this message?